tools/nodered/scripts/lockdown.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Define the path to the Node-RED settings file
+SETTINGS_FILE="$HOME/.node-red/settings.js"
+
+function nodered_restart_if_running
+{
+  if pgrep -x "node-red" >/dev/null; then
+    echo "Outdated Node-RED configuration is still running. Attempting to restart..."
+    node-red-restart
+    echo "Node-RED configuration updated. Please verify that Node-RED is running locally as intended."
+  else
+    echo "Node-RED configuration updated."
```

Comment:
Possibly better message for this spot: "Node-RED not detected running.  Skipping restart"